### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by kevinlee607** [quanxiang-cloud/quanxiang#63] [Announcements]
## What's Changed
* update readme/readme-zh by @hqc19907228 in https://github.com/quanxiang-cloud/quanxiang/pull/40
* uppdate: include new faas and some bug fix by @ding-insper in https://github.com/quanxiang-cloud/quanxiang/pull/43
* feat: support kubernetes 1.22.10 by @kevinlee607 in https://github.com/quanxiang-cloud/quanxiang/pull/50
* fix: builder deploy failed with map values not alowed. by @kevinlee607 in https://github.com/quanxiang-cloud/quanxiang/pull/52
* fix: miss implant dapr pubsub defination by @kevinlee607 in https://github.com/quanxiang-cloud/quanxiang/pull/54
* Per release/v1.1.2 rc1 by @kevinlee607 in https://github.com/quanxiang-cloud/quanxiang/pull/59
* Dapr version by @kevinlee607 in https://github.com/quanxiang-cloud/quanxiang/pull/60
* v2.0.0 realese by @kevinlee607 in https://github.com/quanxiang-cloud/quanxiang/pull/62

**Full Changelog**: https://github.com/quanxiang-cloud/quanxiang/compare/v1.1.0...v2.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/quanxiang-cloud/quanxiang/releases/tag/v2.0.0'>v2.0.0</a>.</em>

**Comment by MartianHY** [comment]
部分模块的源码里未找到Dockerfile，可以提供一下吗？例如：implant、message、auth、letter等

**Reply by kevinlee607** [reply]
这些是ko打包的。https://github.com/ko-build/ko
